scene/gui/tree.cpp

```diff
@@ -2811,6 +2811,20 @@ void Tree::_text_editor_submit(String p_text) {
 	update();
 }
 
+void Tree::_text_editor_changed(String p_text) {
+	if (updating_value_editor) {
+		return;
+	}
+	if (!popup_edited_item) {
+		return;
+	}
+
+	edited_item = popup_edited_item;
+	edited_col = popup_edited_item_col;
```

Comment:
This is necessary because, otherwise `edited_item` and `edited_col` aren't updated properly when writing the node name, which results in this error:

`editor\scene_tree_editor.cpp:845 - Condition "!which" is true.`

If someone knows of a way to address it differently, let me know, please.